templates/gmcq.hbs

```diff
@@ -7,6 +7,9 @@
             <label aria-hidden="true" id="{{../_id}}-{{@index}}-aria" for="{{../_id}}-{{@index}}" class="{{#unless ../_isEnabled}}disabled {{/unless}} component-item-color component-item-text-color component-item-border {{#if _isSelected}}selected{{/if}} a11y-ignore" tabindex="-1">
 
                 <img src="{{_graphic.src}}" data-large="{{_graphic.large}}" data-small="{{_graphic.small}}"/>
+                {{#if _graphic.attribution}}
+                    <div class="gmcq-item-graphic-attribution">{{{_graphic.attribution}}}</div>
```

Comment:
tom has a fair point. originally you had a generic style in the theme. i wanted a copy of the style in each plugin as that's where it's used.

you've renamed the class in such a way that to override all of the attribution divs from the theme you'd have to list them by plugin name.

tom's suggest into to make all the class names the same but nested inside their relevant containers.